marshmallow/fields.py

```diff
@@ -193,7 +193,11 @@ def _validate(self, value):
         kwargs = {}
         for validator in self.validators:
             try:
-                r = validator(value)
+                try:
+                    r = validator(value, field=self)
```

Comment:
`self` should probably passed as a positional argument rather than a keyword argument. That way, users wouldn't be required to name the second argument "field".

Reply:
Users don't have to name the argument, but if they do, it should be field because that's what's being passed in. Unless you have a better name.

Reply:
Constraining the name of the argument seems unnecessary. Different projects might have different naming conventions. For example, I've used `field_obj` to disambiguate field names from field instances.